src/js/dropdown.js

```diff
@@ -250,27 +250,107 @@
 
 // export default Dropdown;
 
+// Focus an element
+// elements - array of dropdown elements
+function focus(elements, focussedIndex) {
+  // Use of % for infinite iterations
+  var index = Math.abs(focussedIndex) % elements.length;
+  if (index >= 0) {
+    elements[index].focus();
+  }
+}
+
+// Normalize itertion counter - Arrow Up
+// Check if the focussedIndex reached a negative number
+// If so, assign it to (elements.length - 1)
+// Else, decrement the counter
+function normalizeNegativeCounter(elements, focussedIndex) {
+  if (focussedIndex <= 0) {
+    return elements.length - 1;
+  } else {
+    return focussedIndex - 1;
+  }
+}
+
 // Alpine Variant
 window.dropdown = function () {
+  const DROPDOWN_SELECTOR = "a.dropdown-item";
```

Comment:
@TheWebDevel dropdown items can be any html tag, not only `a` tags

Can you make sure something like `<div class="dropdown-item">Yellow</div>` is also focusable?

Reply:
Resolved. Please check.